uniffi_bindgen/src/bindings/kotlin/templates/EnumTemplate.kt

```diff
@@ -8,13 +8,24 @@
 {%- if e.is_flat() %}
 
 {%- call kt::docstring(e, 0) %}
+{% match e.variant_discr_type() %}
+{% when None %}
```

Comment:
re more examples above, here it appears that if an enum has literals for each variant but no discriminant type then we just drop the variant literals. That may or may not be the right thing to do, but explicitly spelling it out in the fixture (and the eventual docs) would help make this clearer.